libs/sdk-ui-pivot/src/impl/resizing/tests/columnSizing.test.ts

```diff
@@ -261,11 +262,15 @@ describe("ResizedColumnStore", () => {
         ["for measureless column width on column-only table", ColumnOnlyResultDescriptor, [ColumnOnlyWidth]],
     ];
 
+    function storeSnapshot(store: ResizedColumnsStore) {
```

Comment:
While this certainly works, there is a more idiomatic way of doing this: https://jestjs.io/docs/en/snapshot-testing#property-matchers I am not insisting on this, just thought it might be useful.

Reply:
it's good to know. it does not address this 100% though. because the storeSnapshot also cleans unneeded props so that they do not clutter the snapshot & make its persistence faster. the table descriptor in resized column store is massive and without cleaning it up the tests were timing out.

Reply:
Oh, good point, did not realize that could be an issue.